UPGRADING.md

```diff
@@ -3,6 +3,30 @@ Upgrading Grape
 
 ### Upgrading to >= 0.14.0
 
+#### Changes to behavior of `after` method of middleware on error
+
+The `after` method of the middleware is now called on error.
+The following code would work correctly.
+
+```ruby
+class CustomMiddleware < Grape::Middleware::Base
+  def after
+    puts "Yay!"
+  end
+end
+
+class API < Grape::API
+  use CustomMiddleware
+
+  get { raise StandardError }
+end
+
+mock = Rack::MockRequest.env_for(?/)
```

Comment:
Rewriting is ok. However, the rack response thing should be separated with the error thing because the rack response in `after` is not used in error case.